### PR TITLE
[Website] Add JTBD - WIP

### PR DESCRIPTION
**Pull request by kaitlincarter-hc**
We still need to get design eyes on this and edit the copy, but this is a good preview. 
Screen Shot 2019-11-08 at 1 05 36 PM

### CODE REVIEW

File: website/source/index.html.erb [hunk 1]
```diff
@@ -65,130 +65,75 @@ description: |-
...
-            <img src='/assets/images/consul-connect/svgs/discovery-simple.svg' alt='Service Discovery'>
-            <h3>Service Discovery</h3>
-            <p>Use the service registry to address and discover services across multiple runtime platforms, cloud providers and regions.</p>
+            <h3>Service Deployments</h3>
+            <p>Zero downtime deployments with L7 traffic splitting.</p>
           </div>
-          <div>
-            <a href='/discovery.html' class='g-btn dark-outline'>Learn more</a>
+           <div>
+            <a href='https://learn.hashicorp.com/consul/developer-mesh/consul-splitting' class='g-btn'>Learn more</a>
```
Comment (KFishner) [inline website/source/index.html.erb]: needs utms

File: website/source/index.html.erb [hunk 2]
```diff
@@ -65,130 +65,75 @@ description: |-
...
           <div>
-            <img src='/assets/images/consul-connect/svgs/segmentation-simple.svg' alt='Service Mesh'>
-            <h3>Service Mesh</h3>
-          <p>Service discovery, identity-based authorization, and L7 traffic management abstracted from application code with proxies in the service mesh pattern.</p>
+            <h3>Connect Services across Datacenters</h3>
+            <p>Secure inter-datacenter communication with Connect.</p>
           </div>
           <div>
-            <a href='/mesh.html' class='g-btn dark-outline'>Learn more</a>
+            <a href='https://learn.hashicorp.com/consul/developer-mesh/connect-gateways' class='g-btn'>Learn more</a>
```
Comment (KFishner) [inline website/source/index.html.erb]: utms

File: website/source/index.html.erb [hunk 3]
```diff
@@ -65,130 +65,75 @@ description: |-
...
-    "Session": "1c3f5836-4df4-0e26-6697-90dcce78acd9",
-    "Value": "Zm9v",
-    "Flags": 0,
-    "Key": "deployment",
-    "CreateIndex": 13,
-    "ModifyIndex": 19
-  }
-]</code>
-            </div>
+            <a href='https://learn.hashicorp.com/consul/integrations/haproxy-consul' class='g-btn'>Learn more</a>
```
Comment (KFishner) [inline website/source/index.html.erb]: utms

File: website/source/index.html.erb [hunk 4]
```diff
@@ -65,130 +65,75 @@ description: |-
...
-        </div>
-      </div>
-    </div>
-  </section>
-
-  <section id='use-cases' class='g-section'>
-    <div class='g-container'>
-      <div class='intro'>
-        <h2>Use Cases</h2>
+        <h2>Common Tasks</h2>
```
Comment (KFishner) [inline website/source/index.html.erb]: thoughts on "Tasks Consul can do" or "What Consul can do"

Comment (judithpatudith) [reply website/source/index.html.erb]: Maybe "What can you do with Consul?"

**Comment by judithpatudith** [comment]
How do we plan on using this new section? Are we going to highlight new guides with it, or leave the callouts the same based on the most common use cases? Do we want to make sure that the guides we point to here will always stand alone?  

For example, the Gateway guide has a ton of prereqs currently (we assume that people following it have an environment for testing), but if we're going to point to it from the home page we might want to provide more structure so that people have a starting point.

File: website/source/index.html.erb (hunk 1 above)
Comment (kaitlincarter-hc) [inline website/source/index.html.erb]:
```suggestion
            <a href='https://learn.hashicorp.com/consul/developer-mesh/consul-splitting?utm_source=consul.io&utm_medium=home-page&utm_content=jtbd&utm_term=splitting' class='g-btn'>Learn more</a>
```

File: website/source/index.html.erb (hunk 2 above)
Comment (kaitlincarter-hc) [inline website/source/index.html.erb]:
```suggestion
            <a href='https://learn.hashicorp.com/consul/developer-mesh/connect-gateways?utm_source=consul.io&utm_medium=home-page&utm_content=jtbd&utm_term=gateways' class='g-btn'>Learn more</a>
```

File: website/source/index.html.erb (hunk 3 above)
Comment (kaitlincarter-hc) [inline website/source/index.html.erb]:
```suggestion
            <a href='https://learn.hashicorp.com/consul/integrations/haproxy-consul?utm_source=consul.io&utm_medium=home-page&utm_content=jtbd&utm_term=lb' class='g-btn'>Learn more</a>
```

File: website/source/index.html.erb (hunk 4 above)
Comment (kaitlincarter-hc) [inline website/source/index.html.erb]:
```suggestion
        <h2>What can you do with Consul?</h2>
```

Comment (schersh) [reply website/source/index.html.erb]: Could we phrase this more as a call to action, something like "Put Consul to work?"

**Comment by kaitlincarter-hc** [comment]
@johncowen I'm trying to make the icons slightly bigger, is there a size constraint?

**Pull request merged**